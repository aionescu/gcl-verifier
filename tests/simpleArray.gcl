minIndex(a: [Int], i: Int) -> r: Int {
  assume #a == 10 && i + 1 == 3;
  a[i + 1] = 5;
  a[i] = 8;
  assert a[3] == 5;
}