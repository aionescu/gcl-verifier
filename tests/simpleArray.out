#a == 10 && i + 1 == 3 => (3 == i ->  8 | (3 == i + 1 -> 5 | a[3])) = 5 && True